package-res/resources/web/prompting/PromptPanel.js

```diff
@@ -744,7 +747,18 @@ define(['cdf/lib/Base', 'cdf/Logger', 'dojo/number', 'dojo/i18n', 'common-ui/uti
          */
         parameterChanged: function (param, name, value) {
           if (this.onParameterChanged) {
-            this.onParameterChanged(name, value);
+            if (typeof this.onParameterChanged === 'function') {
+              this.onParameterChanged(name, value);
+            } else {
+              var paramCallback = this.onParameterChanged[name];
+              if (paramCallback) {
+                if (typeof paramCallback === 'function') {
+                  paramCallback(name, value); 
+                } else {
+                  console.log("The parameterChanged callback for '" + name + "' is not a function");
```

Comment:
Shouldn't this instead throw an error? And have a test for that behaviour.

(I was just passing by... nevermind if it doesn't make sense)